### ISSUE TITLE
NavigationViewItem tab control type supports invoke pattern with PaneDisplayMode="Top"

### ISSUE THREAD

**Issue by dennykim** [microsoft/microsoft-ui-xaml#2702]
**Describe the bug**

`NavigationViewItem`s in a `NavigationView` with the `PaneDisplayMode` set to `Top` supports the invoke pattern, which is a violation of [Section 508 502.3.10](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology) (according to Accessibility Insights for Windows).

**Steps to reproduce the bug**

1. Create an app with a `NavigationView` control with `PaneDisplayMode` set to `Top`.
2. Add `NavigationViewItems` in `NavigationView.MenuItems` with any content value: (ex: "Menu Item 1", "Menu Item 2", "Menu Item 3".
3. Run the app with Accessibility Insights.

**Expected behavior**

Expected: Invoke pattern is not supported.

**Screenshots**

**Version Info**

NuGet package version: 

[Microsoft.UI.Xaml 2.4.2]

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| November 2019 Update (18363)  | Yes   |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes   |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Additional context**

**Comment by StephenLPeters** [comment]
@dennykim do you happen to know what happens when you invoke it? I'd guess nothing?

**Comment by StephenLPeters** [comment]
@ojhad FYI

**Comment by dennykim** [comment]
@StephenLPeters invoking selects the item and switches to the selected item. It seems that the functionality is not compromised; only the conflicting accessibility patterns.

**Comment by ranjeshj** [comment]
@anawishnoff  and @chigy as FYI. I believe the invoke and selection happening together is by design. What exactly is the accessibility issue ? I assume we support both invoke and selection patterns on the items.

**Comment by dennykim** [comment]
@ranjeshj according to Accessibility Insights, the accessibility issue is that the invoke pattern shouldn't be supported for this control type (tab item).

**Comment by ranjeshj** [comment]
@YuliKl  @anawishnoff I think we wanted to support both Invoke and Selection. Is that an issue with accessibility ?

**Comment by YuliKl** [comment]
I posed this question on our [internal StackOverflow](https://stackoverflow.microsoft.com/questions/208970) to hopefully receive advice from accessibility experts. I don't currently understand the problem with the existing implementation.

**Comment by dbjorge** [comment]
Hi! I'm a developer on the Accessibility Insights team. I've left [an answer to the internal StackOverflow post](https://stackoverflow.microsoft.com/a/209243/3682) explaining why Accessibility Insights flags this as a violation and what the user impact of it might be.

**Comment by YuliKl** [comment]
Thank you, @dbjorge, for the very helpful answer. I reviewed the [Invoke](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-invoke-control-pattern) and [SelectionItem](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-selectionitem-control-pattern) control patterns, and checked the [Control Pattern Mapping](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/control-pattern-mapping-for-ui-automation-clients) table. 

All this documentation suggests to me that `NavigationViewItems` (in both top and left layouts) should follow the Tab Item precedent from the table and implement only the **Selection Item** pattern and *not* implement the **Invoke** pattern.

@chingucoding, do you have any interest in fixing this issue?

**Comment by chingucoding** [comment]
Of course @YuliKl , would love to! I will look into this.
